solidity/dashboard/README.md

```diff
@@ -55,6 +55,16 @@ truffle exec ./scripts/delegate-tokens.js
 
 * Use metamask with `localhost:8545` to use Ganache test network. Import your first Ganache test account into metamask and you should be able to see the demo data.
 
+### Work with contracts deployed locally
+
```

Comment:
Maybe it's worth mentioning that we need to clone a keep-ecdsa and tbtc repo 🤔.

Reply:
sure, done